static-assets/components/cstudio-forms/controls/numeric-input.js

```diff
@@ -106,11 +106,11 @@ YAHOO.extend(CStudioForms.Controls.numericInput, CStudioForms.CStudioFormField,
 	 * @param el element
 	 */
 	count: function(evt, countEl, el) {
-		var max = this.getAttribute("max") ,
-			min = this.getAttribute("min");
-		if(max && parseInt(this.value) > max){
+		var max = el.maxValue,
+			  min = el.minValue;
+		if(max && max !== '' && parseInt(this.value) > max){
 			this.value = max;
-		} else if(min && parseInt(this.value) < min){
+		} else if(min && min !== '' && parseInt(this.value) < min){
```

Comment:
This will be false on 0. Is that a possibility? is that the intended behaviour?

Reply:
It's a bug, 0 must be considered. Fixing

Reply:
@rart I realized that the 0 comparison won't be false, because at that moment it's a string

Reply:
(0 would be false but if it's a string then it's not `0` 😉)